python/tvm/relax/frontend/onnx_frontend.py

```diff
@@ -552,7 +652,19 @@ def _convert_operator(self, op_name, inputs, attrs, opset):
         """
         convert_map = _get_convert_map(opset)
         if op_name in convert_map:
-            sym = convert_map[op_name](self.bb, inputs, attrs)
+            convert_class = convert_map[op_name]
+            op_function = convert_class.get_converter(opset)
+            # If the op_function is a subclass of RelayOnnxOpConverter then it is a relay op.
+            if (convert_class.__bases__[0] == RelayOnnxOpConverter) or \
+                (hasattr(convert_class.__bases__[0], "__bases__") and convert_class.__bases__[0].__bases__[0] == RelayOnnxOpConverter):
+                relay_inputs = self._relay_input_adapter(inputs)
+                # The op_function might change the inputs to the relay op. Use a copy of the inputs.
+                relay_inputs_copy = [relay_input for relay_input in relay_inputs]
+                # TODO handle params passing
+                relay_output = op_function(relay_inputs_copy, attrs, [])
```

Comment:
a way to make this more explicit while there's still the outstanding TODO is `op_function(relay_inputs_copy, attrs, params=[])`